OJP_JourneySupport.xsd

```diff
@@ -556,6 +561,11 @@
 					<xs:documentation>Whether the result should include accessibility information.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="IncludeTactilePaving" type="xs:boolean" default="false" minOccurs="0">
```

Comment:
Why is an additional parameter `IncludeTactilePaving` a better choice than delivering the paving guidance information when `IncludeAccessibility` is `true`?

Reply:
Because someone that is in a Wheelchair literally hates a route with TactilePaving and prefers a route around it. While someone that is visibility impaired has great preference for it. Two different end users typically thrown in the same group of society having opposite needs.

Reply:
I agree to Stephan. A content filter "IncludeTactilePaving" is not needed here. The content filters define, which kind of information content is returned to the requestor. If he requests accessibility information, he will also get information on tactile pavement. The content filter has nothing to do with the actual routing. This would be influenced by the mobility filter "NoSight".
My conclusion: "NoSight" is sensible, "IncludeTactilePavement" is not.